text/0079-cdk-2.0.md

```diff
@@ -0,0 +1,405 @@
+actions will be taken to understand the causes for slow adoption and identify opportunities to provide additional
+tooling to remove roadblocks and facilitate migration.
+
+[version reporting]: https://docs.aws.amazon.com/cdk/latest/guide/tools.html
+
+## Deprecation of v1
+
+Based on the v1 maintenance plan, feature development is to stop on CDK v1, as only bugs reported against CDK v1 are to
+be fixed. Back-porting of features is not necessary, although the community is still welcome to back-port features.
+
+New GitHub labels will be added to differentiate bugs affecting `v1`, `v2` or both. The triaging process will need to be
+adjusted to include a determination of whether a bug fix against CDK v1 needs forward-porting to v2 or vice-versa. A
+similar provision applies to pull-requests, as only the following should be accepted against v1:
+
+- Features that facilitate migration to v2
+- Features that are back-ported from v2
+
+# AWS CDK Maintenance Policy
+
+Customers of the CDK must be able to rely on the perenity of major version releases. On the other hand, maintaining
+multiple major versions at the same time can be challenging, especially if feature work is expected ot happen on both
+branches. Our goal is to minimize the cost of maintaining multiple major versions at the same time, while enabling
+customers to migrate to a new major version release at their own pace.
+
+To this effect, we define the following life-cycle stages for libraries and tools (later on referenced generically as
+_software_) that are part of the CDK ecosystem:
+
+- **Developer Preview**: During this phase, software is not supported, should not be used in production environments,
+  and are meant for early access and feedback purposes only. It is possible for future releases to introduce breaking
+  changes. Once AWS identifies a release to be a stable product, it may mark it as a _Release Candidate_. _Release
+  Candidates_ are ready for _General Availability_ unless significant bugs emerge, and will receive full AWS support.
+- **Full Support**: During this phase, software is fully supported. AWS will provide regular releases that include new
+  features, support for new services, API updates for existing services, as well as bug and security fixes.
+- **Maintenance Announcement**: AWS will make a public announcement at least 6 months before software enters the
+  **Maintenance** phase. During this period, software will continue to be fully supported. Typically, this announcement
+  happens at the same time as the next major version is introduced.
+- **Maintenance**: During this phase, AWS limits releases to address critical bug fixes and security issues only.
+  Software will no longer receive API updates for new or existing services, or be updated to support new regions. The
+  **Maintenance** phase has a default duration of 12 months, unless otherwise specified as part of the **Maintenance
+  Announcement**.
```

Comment:
okay, this does *not* imply that any bug reported on 1.x will also be fixed on 1.x

Reply:
Only if it's critical / security impacting, yes.